service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/generator/Generator.java

```diff
@@ -102,6 +102,7 @@ private static void determineVersion(String localPomVersion) {
     }
 
     private static void setVersion(String moduleVersion) {
+        moduleVersion = moduleVersion.trim();
```

Comment:
Trim removes just space, but version itself is taken as a command line argument separated by space, hence version here won't have space. Are you sure this fixes the bug?

Reply:
String.trim() removes whitespaces (spaces, new lines etc.), here moduleVersion is coming like "1.0.0\n" or  "1.0.0-SNAPSHOT\n" hance trim solves the problem.